src/containers/Configure/Configure.js

```diff
@@ -9,11 +9,29 @@ const Configure = (props) => {
   const [systemLosses, updateSystemLosses] = useState(null)
   const [tilt, updateTilt] = useState(null)
   const [azimuth, updateAzimuth] = useState(null)
+  const [isLoading, setIsLoading] = useState(false)
+
+useEffect(() =>{
```

Comment:
Interested to see what this stuff will look like when we hook it up to the backend.  From the looks of it, useEffect is just as straight forward and easy to implement as useState